src/PresentationalComponents/Loading/Loading.js

```diff
@@ -0,0 +1,15 @@
+import { Card } from '@patternfly/react-core/dist/js/components/Card/Card';
+import { CardBody } from '@patternfly/react-core/dist/js/components/Card/CardBody';
+import { List } from 'react-content-loader';
+import React from 'react';
+const Loading = () => (
+  <>
```

Comment:
Either use <React.Fragment> or <> but not both, consistency is keeeeeyyyyy tbh tho, here, i dont think yah need it, card is sufficient